pkg/blob/nodeserver.go

```diff
@@ -150,16 +152,20 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		return nil, err
 	}
 
-	if strings.TrimSpace(blobStorageEndPoint) == "" {
+	if strings.TrimSpace(storageEndpointSuffix) == "" {
+		storageEndpointSuffix = "core.windows.net"
+	}
+
+	if strings.TrimSpace(serverAddress) == "" {
 		// server address is "accountname.blob.core.windows.net" by default
-		blobStorageEndPoint = fmt.Sprintf("%s.blob.%s", accountName, d.cloud.Environment.StorageEndpointSuffix)
```

Comment:
I think we need to keep `d.cloud.Environment.StorageEndpointSuffix` if `storageEndpointSuffix` is empty, that will not break azure stack test @songjiaxun